qadence/parameters.py

```diff
@@ -204,7 +205,15 @@ def heaviside_func(x: Tensor, _: Any) -> Tensor:
         return res
 
     extra_funcs = {sympy.core.numbers.ImaginaryUnit: 1.0j, sympy.Heaviside: heaviside_func}
-    return SymPyModule(expressions=[sympy.N(expr)], extra_funcs=extra_funcs)
+    return torchSympyModule(expressions=[sympy.N(expr)], extra_funcs=extra_funcs)
+
+
+def make_differentiable(expr: Expr, engine: Engine) -> torchSympyModule | JaxSympyModule:
+    if engine == Engine.JAX:
+        return JaxSympyModule(expr)
+    else:
+        extra_funcs = {sympy.core.numbers.ImaginaryUnit: 1.0j}
+        return torchSympyModule(expressions=[sympy.N(expr)], extra_funcs=extra_funcs)
```

Comment:
this seems to be part of the `Heaviside` docs failure

Reply:
yeah i noticed, will fix later. but there was something else failing in the docs